lab_1/main.py

```diff
@@ -12,8 +13,22 @@ def tokenize(text: str) -> list:
     e.g. text = 'The weather is sunny, the man is happy.'
     --> ['the', 'weather', 'is', 'sunny', 'the', 'man', 'is', 'happy']
     """
-    pass
-
+    if type(text)==str:
```

Comment:
Usually, there are checks for input arguments and function body - separate.
And body function is on the same level as checks.